### PR TITLE
Update name_cleaners.py to use os separator

### PR DESCRIPTION
**Pull request by mgoldey**

### CODE REVIEW

File: asrtoolkit/file_utils/name_cleaners.py [hunk 1]
```diff
@@ -43,7 +44,7 @@ def replace_char(input_char, file_name):
                 "check to make sure your audio files and transcript files match"
             )
         return "/".join(
```
Comment (cbrochtrup) [inline asrtoolkit/file_utils/name_cleaners.py]: This should be `os.sep` too, unless that won't work with join for some reason.

Comment (mgoldey) [inline asrtoolkit/file_utils/name_cleaners.py]:
```suggestion
        return os.sep.join(
```

Comment (mgoldey) [inline asrtoolkit/file_utils/name_cleaners.py]:
```suggestion
        return os.sep.join(
```

File: asrtoolkit/file_utils/name_cleaners.py [hunk 2]
```diff
@@ -16,8 +17,8 @@ def get_extension(file_name):
 def basename(file_name):
     """
     Returns basename of a file without the preceding directory
-  """
-    return file_name.split("/")[-1]
+    """
+    return file_name.split(os.sep)[-1]
```
Comment (mgoldey) [inline asrtoolkit/file_utils/name_cleaners.py]:
```suggestion
    return os.path.basename(file_name)
```

**Pull request merged**